src/testRunner/unittests/tsserver/events/watchEvents.ts

```diff
@@ -93,8 +93,8 @@ describe("unittests:: tsserver:: events:: watchEvents", () => {
     function updateFileOnHost(session: TestSession, file: string, log: string, content?: string) {
         // Change b.ts
         session.logger.log(`${log}: ${file}`);
-        if (content) session.host.appendFile(file, content);
-        else session.host.writeFile(file, session.host.readFile("/user/username/projects/myproject/a.ts")!);
+        if (content && session.host.fileExists(file)) session.host.appendFile(file, content);

```

Comment:
I know this is just testing, but is there technically an issue here with if `content` is an empty string?

Reply:
yeah but its for testing so just simplified it